spras/config.py

```diff
@@ -69,6 +70,8 @@ def __init__(self, raw_config):
         self.unpack_singularity = False
         # A dictionary to store configured datasets against which SPRAS will be run
         self.datasets = None
+        # A dictionary to store configured gold standard data against ouptut of SPRAS runs
+        self.gold_standard = None
```

Comment:
```suggestion
        self.gold_standards = None
```

Does this need to be plural? There can be multiple gold standards?